src/implementations/twig/components/social-media-follow/social-media-follow.story.js

```diff
@@ -9,9 +9,11 @@ import notes from './README.md';
 // Preserve original data.
 const dataHorizontal = { ...specs };
 const dataVertical = { ...specs, variant: 'vertical' };
+const others = { ...specs.links[5] };
```

Comment:
it may be safer to get the last item of the array, instead of a specific number (if we have to add or remove an item in 6 months or so)

Reply:
Yep, makes sense, although i discovered that there is an "editorial" limit set to 6 items, that's the reason why i removed the mastodon icon, it was astrid requesting that.. :)